app/constant/Layout.ts

```diff
@@ -17,4 +17,7 @@ const screen = {
   height,
 };
 
-export { screen, isSmallDevice, PADDING_HORIZONTAL, SPACING };
+const ANDROID_TAB_BAR_HEIGHT = 100;
```

Comment:
Please use `useBottomTabBarHeight`, see [documentation](https://reactnavigation.org/docs/bottom-tab-navigator/#usebottomtabbarheight).

Reply:
@PierreBresson updated the approach to use different hook.